client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dataviews/index.tsx

```diff
@@ -95,47 +95,47 @@ const SitesDataViews = ( { data, isLoading }: Props ) => {
 		},
 		{
 			id: 'stats',
-			header: 'STATS',
+			header: <span>STATS</span>,
 			getValue: () => 'Stats status',
```

Comment:
Should this (and other counterparts) be translated?

Reply:
This is preexisting, but as far as I know we do not translate product names.

Reply:
I'm referring to line 99 ("Stats status"). cc: @cleacos as well for consideration.

Reply:
That comes from the prototype experiment, but I still don't know where that field is used, so I didn't make it translatable until we discovered where it could be used. Probably, we can leave it empty. I'll add another issue to check it out and if so, to leave it empty.

Reply:
Follow-up issue: https://github.com/Automattic/jetpack-manage/issues/310